### PR TITLE
First configuration for tests

### PR DESCRIPTION
**Pull request by anaschwendler**
Tries to solve #3.

What I did?
Using as the base [this project](https://github.com/codingjoe/django-stdimage) I've created two files to serve as set up for the test environment.

[`setup.cfg`](https://github.com/codingjoe/django-stdimage/blob/305b7f5ca5aa0d7785b2fdc7514dfb3292b28ec1/setup.cfg) to set up pytest django settings file.

[`tests/settings.py`](https://github.com/codingjoe/django-stdimage/blob/master/tests/settings.py) with a basic setup to serve as settings file for the test environment.

What is happening now?
I don't do python in a long time 😅 but it seems to throw and error related to `__init__.py`
image

### CODE REVIEW

File: tests/settings.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -*- encoding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+import tempfile
```
Comment (cuducos) [inline tests/settings.py]: Imported but unnused? If so, we can delete this line ; )

File: tests/settings.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- encoding: utf-8 -*-
+from __future__ import unicode_literals
```
Comment (cuducos) [inline tests/settings.py]: As we're using Python 3, I think we don't need these lines (1-2) ; )

File: tests/settings.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.sites',
+    'public_admin',
+    'tests'
+)
+
+DEFAULT_FILE_STORAGE = 'tests.storage.MyFileSystemStorage'
```
Comment (cuducos) [inline tests/settings.py]: Do we really need it? I mean, [Django already has a default value to use if this variable is unset](https://docs.djangoproject.com/en/3.0/ref/settings/#std:setting-DEFAULT_FILE_STORAGE) and I don't get why we should set it to an non-existent object ; )

**Review by cuducos — COMMENTED** [review]
Great progress :sparkles:  With that we tick the 1st checkbox of #3 :purple_heart: 

Regarding the error message, I think it's a fair message: maybe the `DummyUser` expects `public_apps` and `pulic_models` to be passed as keyword arguments, not simple arguments.

File: tests/settings.py (hunk 3 above)
Comment (anaschwendler) [reply tests/settings.py]: Yes, in the end, I added those extra things thinking that they might be useful, but makes sense to remove those lines.

**Pull request merged**